syntax = "proto2";

enum MsgType {
	REQ_INFERENCE = 1;
	REQ_CLOSE     = 2;

	RSP_INFERENCE = 3;
	RSP_CLOSE			= 4;
}

message InferenceReqProto {
	required uint32 req_id = 1;
	required uint32 model_id = 2;
	required uint32 batch_size = 3;
}

message InferenceRspProto {
	required uint32 req_id = 1;
	required int32 dummy = 2;
}

message CloseReqProto {
	required uint32 req_id = 1;
	required int32 dummy = 2;
}

message CloseRspProto {
	required uint32 req_id = 1;
	required int32 dummy = 2;
}
